react/react1/week1/todo-list/src/componate/TodoList.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+function TodoList(props) {
+    return <div>
+        <ul>
+            <li>{props.description}</li>
```

Comment:
you can also use:
```suggestion
            <li>{todoItem.description}</li>
```
it would be cleaner